arconia-bom/build.gradle

```diff
@@ -20,8 +20,9 @@ dependencies {
         api project(":arconia-dev:arconia-dev-services:arconia-dev-services-core")
         api project(":arconia-dev:arconia-dev-services:arconia-dev-services-lgtm")
         api project(":arconia-dev:arconia-dev-services:arconia-dev-services-mysql")
-        api project(":arconia-dev:arconia-dev-services:arconia-dev-services-ollama")
+        api project(":arconia-dev:arconia-dev-services:arconia-dev-services-mariadb")
```

Comment:
I like the idea of grouping the DB-related services in the docs, but in here and in the `settings.gradle` I would keep the alphabetical sorting. What do you think?

Reply:
Done